pkgs/development/python-modules/pyarrow/default.nix

```diff
@@ -106,6 +97,9 @@ buildPythonPackage rec {
   '';
 
   pytestFlagsArray = [
+    # A couple of tests are missing fixture imports, luckily pytest offers a
+    # clean solution.
+    "--fixtures pyarrow/tests/conftest.py"
```

Comment:
The `--fixtures` flag take no argument option, but only set `showfixtures` variable as `True`.

https://github.com/pytest-dev/pytest/blob/1d7d63555e431d4562bcacbdc97038b0613d20ba/src/_pytest/fixtures.py#L1448-L1456

`pytest --fixtures pyarrow/tests/conftest.py` is essentially equivalent to `pytest pyarrow/tests/conftest.py --fixtures`, which will only collect and run tests inside `pyarrow/tests/conftest.py`. This seems contradicting to the `--deselect` flags below.

Is that what you intend to do with this line?

Reply:
I don't remember, but according to the comment probably not. Did you try to build it without that argument?

Reply:
I'm seeing test failures/errors in
- pyarrow/tests/test_fs.py
- pyarrow/tests/test_orc.py
- pyarrow/tests/test_substrait.py
- pyarrow/tests/test_types.py
- pyarrow/tests/test_udf.py

Most tests in the above files still pass.

Reply:
Please open a PR in case this is urgent for you. Otherwise I would fix it with the next version bump.

Reply:
Done in https://github.com/NixOS/nixpkgs/pull/426611.